x/incentives/client/cli/query.go

```diff
@@ -384,31 +386,77 @@ $ %s query incentives rewards-estimation
 				return err
 			}
 
-			lockIdStrs := strings.Split(lockIdsCombined, ",")
-			lockIds := []uint64{}
-			for _, lockIdStr := range lockIdStrs {
-				lockId, err := strconv.ParseUint(lockIdStr, 10, 64)
+			var res *lockuptypes.AccountLockedLongerDurationResponse
+			if owner != "" {
+				queryClientLockup := lockuptypes.NewQueryClient(clientCtx)
```

Comment:
Consider moving all this logic inside the if branch + `ownerLocks` creation to this branch of the following `else if`:
```
else if lockIdsCombined == "" {
    ....
}
```

They are logically equivalent and `ownerLocks` are only needed there since we are copying them all to `lockIds`

Reply:
I think this comment is related to: https://github.com/osmosis-labs/osmosis/pull/1985/files#r915441993

Something feels off about how and where we are using `ownerLocks`